buildenv/jenkins/common/variables-functions

```diff
@@ -207,6 +216,17 @@ def set_test_variables() {
     TEST_DEPENDENCIES_JOB_NAME = VARIABLES.test_dependencies_job_name
 }
 
+/*
+* Set TESTS_TARGETS, indicating the level of testing. 
+*/
+def set_test_targets() {
+    TESTS_TARGETS = params.TESTS_TARGETS
+    if (!TESTS_TARGETS) {
```

Comment:
`TEST_TARGETS` will either be `null` (no param defined), blank (param defined but no value), or will have a value. I would like to see a test case for each to prove this `if` condition works as expected.
It is different than how we do it in [variables-functions](https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/common/variables-functions).

Especially since I got burned by my change in #1320

Reply:
In groovy, both null and the empty string evaluate to false
if ((TESTS_TARGETS == null) || ((TESTS_TARGETS == '')) {...} thus the contraption (!TESTS_TARGETS){...}

Reply:
Right. I made the mistake of wrapping quotes around `null` in my change. I think this is correct.